super_sim_commands.go

```diff
@@ -0,0 +1,146 @@
+package twilio
+
+import (
+	"context"
+	"net/url"
+)
+
+const smsCommandsPathPart = "SmsCommands"
+const commandsPathPart = "Commands"
+
+// SuperCommandsService handles API requests for all SuperSim Commands
+type SuperCommandsService struct {
```

Comment:
The naming here is to differentiate from the `Wireless` `CommandService`. Open to other ideas. `SuperSimCommandService` felt a bit long...